lib/uplink/ext/tests.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+//go:generate go run .
+//go:generate go test .
```

Comment:
Is this a leftover? `go generate` shouldn't run tests.

Reply:
It was convenient during development to have a make target that built the .so and ran the c api tests. Currently, the `test-bindings` does this by calling generate on this file. What would you suggest doing instead?